assets/_sass/_custom.scss

```diff
@@ -1184,7 +1184,7 @@ $tiniest: new-breakpoint(max-width 390px);
 	}
```

Comment:
The CSS you edited all looks fine here. I do see an existing problem of deeply nesting. It might help long-term style maintenance by keeping the css modular. So you would have one _footer.scss file to keep this stuff in like this:

```css
.footer
.footer-container
.footer-link
...
``` 

This isn't necessary now, but something to consider if you do refactoring of styles in the future.